torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1732,8 +1729,9 @@ void ProcessGroupNCCL::watchdogHandler() {
             ", last completed NCCL work: ",
             pgStatus_->lastCompletedSeq,
             ".");
-        // try to dump flight records if exception happens.
-        // Flight recorder behavior should be independent of desync Debug
+        // try to notify other ranks via global TCPStore to dump the flight
+        // recorder if exception happens. Flight recorder behavior is
+        // independent of desync Debug.
         if (dumpOnException_) {
```

Comment:
nit: some folks got confused by 'exception' here, iirc.  it is actually timeout or exception, isn't it?

Reply:
Sure, I also changed the variable name, lol